translations/hu/get5.phrases.txt

```diff
@@ -2,346 +2,502 @@
     }
     "CaptainLeftDuringMapSelection"
     {
-        "hu"            "A csapatkapitány kilépett a pálya választás alatt. A pálya választás szüneteltetve."
+        "hu"        "A csapatkapitány kilépett a pálya választás alatt. A pálya választás szüneteltetve."
     }
     "ReadyToResumeMapSelection"
     {
-        "hu"            "Használd a {1} parancsot, hogy folytasd a pálya választást."
+        "hu"        "Használd a {1} parancsot, hogy folytasd a pálya választást."
     }
     "MatchConfigLoadedInfoMessage"
     {
-        "hu"            "Mérkőzés beállítás betöltödött."
+        "hu"        "Mérkőzés beállítás betöltödött."
     }
     "ExitCoachSlotHelp"
     {
-        "hu"            "Használd a {1} parancsot, hogy kilépj az edzői poziccióból."
+        "hu"        "Használd a {1} parancsot, hogy kilépj az edzői poziccióból."
     }
     "EnterCoachSlotHelp"
     {
-        "hu"            "Használd a {1} parancsot, hogy a csapatod edzője legyél."
+        "hu"        "Használd a {1} parancsot, hogy a csapatod edzője legyél."
     }
     "MoveToCoachInfoMessage"
     {
-        "hu"            "Mivel a csapatod megtelt, így edzői poziccióba lettél átmozgatva."
+        "hu"        "Mivel a csapatod megtelt, így edzői poziccióba lettél átmozgatva."
+    }
+    "CannotLeaveCoachingTeamIsFull"
+    {
+        "hu"        "Nem hagyhatod el az edzői pozicciód, ameddig a csapatod teljes."
+    }
+    "CoachingNotEnabled"
+    {
+        "hu"        "Az edzői pozicció nincs engedélyezve.Be kell, hogy állítsd a {1} parancsot 1-es értékre."
+    }
+    "PlayerIsCoachingTeam"
+    {
+        "hu"        "{1} ezdző a {2} csapatnak."
+    }
+    "CanOnlyCoachDuringWarmup"
+    {
+        "hu"        "Csak bemelegítés alatt válthatsz vagy hagyhatod el, az edzői pozicciót."
+    }
+    "AllCoachSlotsFilledForTeam"
+    {
+        "hu"        "Minden edzői pozicció ({1}) megtelt a csapatban."
     }
     "ReadyTag"
     {
-        "hu"            "[FELKÉSZÜLT]"
+        "hu"        "[FELKÉSZÜLT]"
     }
     "NotReadyTag"
     {
-        "hu"            "[NEM ÁLL KÉSZEN]"
+        "hu"        "[NEM ÁLL KÉSZEN]"
+    }
+    "MapSelectionTeamChatOnly"
+    {
+        "hu"        "Csak csapat csevegőben írhatsz, pályaválasztás alatt."
+    }
+    "MapSelectionInvalidMap"
+    {
+        "hu"        "{1} nem érvényes pálya."
+    }
+    "MapSelectionPickMapHelp"
+    {
+        "hu"        "Használd a {1} <pálya>, hogy pályát kiválaszd."
+    }
+    "MapSelectionBanMapHelp"
+    {
+        "hu"        "Használd a {1} <pálya>, hogy pályát eltávolítsd."
+    }
+    "MapSelectionPickSideHelp"
+    {
+        "hu"        "Használd a {1} vagy a {2} az oldalválasztáshoz."
+    }
+    "MapSelectionAnnounceCaptains"
+    {
+        "hu"        "Csapatkapitánya a {1}: {2}"
+    }
+    "MapSelectionBan"
+    {
+        "hu"        "Eltávolít"
+    }
+    "MapSelectionPick"
+    {
+        "hu"        "Választ"
```

Comment:
This should be `VÁLASZT` - we use caps for this in all languages as the word is colored green inside `MapSelectionTurnToPick`.